demisto_sdk/commands/update_release_notes/update_rn.py

```diff
@@ -458,12 +457,11 @@ def build_rn_template(self, changed_items: dict) -> str:
         rn_template_as_dict: dict = {}
         if self.is_force:
             rn_string = self.build_rn_desc(content_name=self.pack)
-        for content_name, data in sorted(changed_items.items(),
-                                         key=lambda x: RN_HEADER_BY_FILE_TYPE[x[1].get('type', '')] if x[1].get('type')
+        for (content_name, _type), data in sorted(changed_items.items(),
```

Comment:
Add a comment explaining the structure of 'x'

Reply:
Done